e2e/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
Should we use prettier like all our other packages?

Reply:
I copied this from portal frontend, so does it need to be updated as well?

Reply:
Not sure, I have no idea what this file does.

Reply:
It configures the code style for editors. It's set up the same for portal and ALCS frontends, not for services. Perhaps it is part of Angular scaffolding?

For now I'll leave it, but I'll install prettier as a dev dep and include `.prettierrc` as well.

Reply:
Added prettier and eslint